keybert/llm/_openai.py

```diff
@@ -178,15 +178,15 @@ def extract_keywords(self, documents: List[str], candidate_keywords: List[List[s
                     response = chat_completions_with_backoff(self.client, **kwargs)
                 else:
                     response = self.client.chat.completions.create(**kwargs)
-                keywords = response.choices[0].message.content.strip()
+                keywords = response.choices[0].text.strip()
```

Comment:
Shouldn't this still be `message.content`? I believe the way you access a `chat.completions` is different from a `.completions`.

Reply:
@MaartenGr you are right, my bad. I fixed it, now `.text` has been applied only to non-chat models.

Reply:
@MaartenGr any update on that?

Reply:
Sorry for the late reply! Looking at the [OpenAI documentation](https://github.com/openai/openai-python/discussions/742) for migration it seems indeed that `.text` should be used for non-completion tasks, so models like "curie" I believe. However, looking at the code, it seems that `chat=False` by default even though a chat model is used `"gpt-3.5-turbo-instruct"`. 

Looking back at your initial example it seems that you only needed to set `chat=True` to make sure your code works:

```python
import openai
from keybert.llm import OpenAI
from keybert import KeyLLM, KeyBERT

# Create your LLM 
client = openai.OpenAI(api_key="sk-XXXXXXXXX")
llm = OpenAI(client, prompt=CUSTOM_PROMPT, chat=True)

# Load it in KeyLLM
kw_model = KeyBERT(llm=llm)

# Extract keywords
keywords = kw_model.extract_keywords(documents)
```

Have you tried with this PR to see if everything works when comparing a chat vs a non-chat model together with the `chat` parameter?

Reply:
@MaartenGr 

If I set `chat=True`, I got this error:

`NotFoundError: Error code: 404 - {'error': {'message': 'This is not a chat model and thus not supported in the v1/chat/completions endpoint. Did you mean to use v1/completions?', 'type': 'invalid_request_error', 'param': 'model', 'code': None}}`.

`"gpt-3.5-turbo-instruct"` models are non-chat one according to [documentation](https://platform.openai.com/docs/models/model-endpoint-compatibility#:~:text=gpt%2D3.5%2Dturbo-,/v1/completions,-(Legacy)).

Reply:
Ah, that's right, thanks for sharing!